src/link/MachO/Dylib.zig

```diff
@@ -207,7 +207,7 @@ fn readLoadCommands(
                 }
             },
             else => {
-                log.debug("Unknown load command detected: 0x{x}.", .{cmd.cmd()});
+                log.debug("Unknown load command detected: 0x{x}.", .{@enumToInt(cmd.cmd())});
```

Comment:
Excellent example here. I'm pretty sure currently this would print as `0xSYMSEG` or whatever because they assumed it would print the integer of the enum but that's not the case. Clearly we want a number here, so now it'll print as `0x3`.